ui/app/components/ui/identicon/blockieIdenticon/blockieIdenticon.component.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useRef, useState } from 'react'
```

Comment:
This was my first time using React hooks! Also the first time I can recall getting to use the shorthand React Fragment syntax.